apis/v1alpha1/webhook.go

```diff
@@ -82,13 +84,34 @@ func (r *CrdbCluster) Default() {
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type.
 func (r *CrdbCluster) ValidateCreate() error {
 	webhookLog.Info("validate create", "name", r.Name)
+	var errors []error
```

Comment:
why do we need to have a slice of errors if there can be at most one error?

Reply:
Right now, there are minimal checks in webhook so returning single error is fine.
But as we keep on adding more validation checks in webhook, ideally we should validate the complete CR object and return the combined error in one go instead of returning one by one error in each iteration by the user.
Hence added slice of errors here.